### ISSUE TITLE
Documentation: Databases

### ISSUE THREAD

**Issue by greenaar** [woodpecker-ci/woodpecker#4368]
### Clear and concise description of the problem

The sqlite database woodpecker uses by default (especially with helm installs) does not handle multiple incoming webhooks well.

Case in point, if I made multiple commits to different repos within a minute, I'd get some tasks failing to start with 'Database is locked' in the server log.  They would not show up as pending jobs.

Running fewer, or re-running them would work without issues, provided concurrency was low.

### Suggested solution

It would be valuable to include a note in the documentation encouraging setup with an external database for anything beyond an initial prototype - I would even say that starting with a external database saves the efforts required in later migration and the associated hassles.

### Alternative

Alternatively, the helm chart/docker examples could come with mysql/postgresql configured out of the box, but that puts the onus on your side, as well as increasing the installation requirements.

### Validations

- [X] Checked that the feature isn't part of the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

**Comment by anbraten** [comment]
That might also be sth we can try to solve in code 😅  (Just a short brainstorm from what I had in mind) I guess this is related to the way we create pipeline entries. To be able to have an increasing pipeline number we first do sth like `select count(id) from pipelines where repo=...` and probably lock the table til we created that entry.

**Comment by zc-devs** [comment]
SQLite evolves. Now it's better than before.
Grafana works well with SQLite. So, problem in Woodpecker.

> Database is locked

#1905, #2827

**Comment by yannickincyberatlantis** [comment]
Hi, today I performed a sqlite -> Postgres migration for my Woodpecker instance. The steps I performed are below:

TL;DR: Initialize Postgres database with Woodpecker, clean it (server_configs & forges tables), use pgloader in data only mode to copy data from sqlite to Postgres.

1. Stop Woodpecker server
2. For safety: make a backup of the sqlite database
3. Create a Postgres database + user for Woodpecker
4. Update your Woodpecker server config to point to Postgres
5. Start Woodpecker server once to let it initialise the Postgres database. Make sure to check the logs of Woodpecker server to see if everything is going well. If it is, you should see this:
 
	```
	Jan 31 14:46:03 vcs woodpecker-server[676731]: {"level":"info","time":"2025-01-31T14:46:03Z","message":"starting Woodpecker server with version '3.0.1'"}
	Jan 31 14:46:03 vcs woodpecker-server[676731]: {"level":"info","time":"2025-01-31T14:46:03Z","message":"starting http server ..."}
	Jan 31 14:46:03 vcs woodpecker-server[676731]: {"level":"info","time":"2025-01-31T14:46:03Z","message":"starting cron service ..."}
	Jan 31 14:46:03 vcs woodpecker-server[676731]: {"level":"info","time":"2025-01-31T14:46:03Z","message":"starting grpc server ..."}
	```
6. Stop Woodpecker server again
7. Open a SQL console on Postgres (`su - postgres -c psql` or whatever works for your system) and **in the woodpecker database** and run these two queries:

	```SQL
	DELETE FROM server_configs;
	DELETE FROM forges;
	```

	Then close the console again
8. Drop the `gated` column from the `repos` table in the Woodpecker server sqlite database (make sure that you are *not* modifying your only copy). You can use this SQL statement for that:
	
	```SQL
	ALTER TABLE repos DROP COLUMN gated;
	```
	
9. Install `pgloader`. Check the docs here: https://pgloader.readthedocs.io/en/latest/install.html
10. Create a file called `woodpecker.load` and give it these contents:
	
	```
	LOAD DATABASE
        FROM sqlite:///path/to/your/database/woodpecker.db
        INTO pgsql://woodpecker:PASSWORDHERE@localhost/woodpecker
        WITH data only
        ;
	```
11. Run pgloader: `pgloader woodpecker.load`

	You should see a bunch of warnings but no errors (see further below for the output I have). If you only see warnings, fire up Woodpecker server again and you *should* have a working setup.

So far (for about two hours) it has been running fine. Use at your own risk etc.

Notes:

- The `server_configs` and `forges` tables are cleared from the Postgres database because these tables may be pre-filled by Woodpecker. Since we're going to import everything from our sqlite database anyway we just nuke it.
- The `gated` column exists for me in sqlite but did not exist in Postgres. I don't know if this column was used at a certain point in time but not any more, or what's up. Either way, pgloader throws errors if a column exists in the sqlite database but not in the Postgres one, so we delete it.

Pgloader output:

```
❯ ./pgloader woodpecker.load
2025-01-31T16:14:00.007000+01:00 LOG pgloader version "3.6.9"
2025-01-31T16:14:00.008000+01:00 LOG Data errors in '/tmp/pgloader/'
2025-01-31T16:14:00.008000+01:00 LOG Parsing commands from file #P"/home/USERNAME/Downloads/pgloader-bundle-3.6.9/bin/woodpecker.load"
2025-01-31T16:14:00.074001+01:00 LOG Migrating from #<SQLITE-CONNECTION sqlite:///home/USERNAME/Downloads/woodpecker.sqlite3 {10063A2FA3}>
2025-01-31T16:14:00.074001+01:00 LOG Migrating into #<PGSQL-CONNECTION pgsql://woodpecker@localhost:5432/woodpecker {10063A3023}>
2025-01-31T16:14:00.171001+01:00 WARNING Source column "public"."migration"."id" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."migration"."id".
2025-01-31T16:14:00.171001+01:00 WARNING Source column "public"."migration"."description" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."migration"."description".
2025-01-31T16:14:00.171001+01:00 WARNING Source column "public"."agents"."name" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."agents"."name".
2025-01-31T16:14:00.171001+01:00 WARNING Source column "public"."agents"."token" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."agents"."token".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."agents"."platform" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."agents"."platform".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."agents"."backend" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."agents"."backend".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."agents"."capacity" is casted to type "bigint" which is not the same as "integer", the type of current target database column "public"."agents"."capacity".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."agents"."version" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."agents"."version".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."agents"."no_schedule" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."agents"."no_schedule".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."agents"."custom_labels" is casted to type "text" which is not the same as "json", the type of current target database column "public"."agents"."custom_labels".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."author" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."author".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."event" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."event".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."status" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."status".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."errors" is casted to type "text" which is not the same as "json", the type of current target database column "public"."pipelines"."errors".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."deploy" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."deploy".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."commit" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."commit".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."branch" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."branch".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."ref" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."ref".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."refspec" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."refspec".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."title" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."title".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."sender" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."sender".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."avatar" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."avatar".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."email" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."email".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."forge_url" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."forge_url".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."reviewer" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."reviewer".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."additional_variables" is casted to type "text" which is not the same as "json", the type of current target database column "public"."pipelines"."additional_variables".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."pr_labels" is casted to type "text" which is not the same as "json", the type of current target database column "public"."pipelines"."pr_labels".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."is_prerelease" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."pipelines"."is_prerelease".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."deploy_task" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."pipelines"."deploy_task".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."pipelines"."from_fork" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."pipelines"."from_fork".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."configs"."hash" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."configs"."hash".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."configs"."name" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."configs"."name".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."log_entries"."line" is casted to type "bigint" which is not the same as "integer", the type of current target database column "public"."log_entries"."line".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."log_entries"."type" is casted to type "bigint" which is not the same as "integer", the type of current target database column "public"."log_entries"."type".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."perms"."pull" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."perms"."pull".
2025-01-31T16:14:00.172001+01:00 WARNING Source column "public"."perms"."push" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."perms"."push".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."perms"."admin" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."perms"."admin".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."steps"."uuid" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."steps"."uuid".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."steps"."pid" is casted to type "bigint" which is not the same as "integer", the type of current target database column "public"."steps"."pid".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."steps"."ppid" is casted to type "bigint" which is not the same as "integer", the type of current target database column "public"."steps"."ppid".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."steps"."name" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."steps"."name".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."steps"."state" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."steps"."state".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."steps"."failure" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."steps"."failure".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."steps"."exit_code" is casted to type "bigint" which is not the same as "integer", the type of current target database column "public"."steps"."exit_code".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."steps"."type" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."steps"."type".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."registries"."address" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."registries"."address".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."registries"."username" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."registries"."username".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."secrets"."name" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."secrets"."name".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."secrets"."images" is casted to type "text" which is not the same as "json", the type of current target database column "public"."secrets"."images".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."secrets"."events" is casted to type "text" which is not the same as "json", the type of current target database column "public"."secrets"."events".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."tasks"."id" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."tasks"."id".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."tasks"."labels" is casted to type "text" which is not the same as "json", the type of current target database column "public"."tasks"."labels".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."tasks"."dependencies" is casted to type "text" which is not the same as "json", the type of current target database column "public"."tasks"."dependencies".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."tasks"."run_on" is casted to type "text" which is not the same as "json", the type of current target database column "public"."tasks"."run_on".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."tasks"."dependencies_status" is casted to type "text" which is not the same as "json", the type of current target database column "public"."tasks"."dependencies_status".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."users"."forge_remote_id" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."users"."forge_remote_id".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."users"."login" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."users"."login".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."users"."email" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."users"."email".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."users"."avatar" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."users"."avatar".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."users"."admin" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."users"."admin".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."users"."hash" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."users"."hash".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."server_configs"."key" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."server_configs"."key".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."server_configs"."value" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."server_configs"."value".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."crons"."name" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."crons"."name".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."crons"."schedule" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."crons"."schedule".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."crons"."branch" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."crons"."branch".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."redirections"."repo_full_name" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."redirections"."repo_full_name".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."workflows"."pid" is casted to type "bigint" which is not the same as "integer", the type of current target database column "public"."workflows"."pid".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."workflows"."name" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."workflows"."name".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."workflows"."state" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."workflows"."state".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."workflows"."platform" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."workflows"."platform".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."workflows"."environ" is casted to type "text" which is not the same as "json", the type of current target database column "public"."workflows"."environ".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."workflows"."axis_id" is casted to type "bigint" which is not the same as "integer", the type of current target database column "public"."workflows"."axis_id".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."orgs"."name" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."orgs"."name".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."orgs"."is_user" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."orgs"."is_user".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."orgs"."private" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."orgs"."private".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."forges"."type" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."forges"."type".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."forges"."url" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."forges"."url".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."forges"."client" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."forges"."client".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."forges"."client_secret" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."forges"."client_secret".
2025-01-31T16:14:00.173001+01:00 WARNING Source column "public"."forges"."skip_verify" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."forges"."skip_verify".
2025-01-31T16:14:00.174001+01:00 WARNING Source column "public"."forges"."oauth_host" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."forges"."oauth_host".
2025-01-31T16:14:00.174001+01:00 WARNING Source column "public"."forges"."additional_options" is casted to type "text" which is not the same as "json", the type of current target database column "public"."forges"."additional_options".
2025-01-31T16:14:00.174001+01:00 WARNING Source column "public"."repos"."forge_remote_id" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."forge_remote_id".
2025-01-31T16:14:00.174001+01:00 WARNING Source column "public"."repos"."owner" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."owner".
2025-01-31T16:14:00.174001+01:00 WARNING Source column "public"."repos"."name" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."name".
2025-01-31T16:14:00.174001+01:00 WARNING Source column "public"."repos"."full_name" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."full_name".
2025-01-31T16:14:00.174001+01:00 WARNING Source column "public"."repos"."avatar" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."avatar".
2025-01-31T16:14:00.174001+01:00 WARNING Source column "public"."repos"."forge_url" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."forge_url".
2025-01-31T16:14:00.174001+01:00 WARNING Source column "public"."repos"."clone" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."clone".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."clone_ssh" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."clone_ssh".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."branch" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."branch".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."pr_enabled" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."repos"."pr_enabled".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."visibility" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."visibility".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."private" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."repos"."private".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."active" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."repos"."active".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."allow_pr" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."repos"."allow_pr".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."config_path" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."config_path".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."hash" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."hash".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."cancel_previous_pipeline_events" is casted to type "text" which is not the same as "json", the type of current target database column "public"."repos"."cancel_previous_pipeline_events".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."allow_deploy" is casted to type "bigint" which is not the same as "boolean", the type of current target database column "public"."repos"."allow_deploy".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."require_approval" is casted to type "text" which is not the same as "character varying", the type of current target database column "public"."repos"."require_approval".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."trusted" is casted to type "text" which is not the same as "json", the type of current target database column "public"."repos"."trusted".
2025-01-31T16:14:00.175001+01:00 WARNING Source column "public"."repos"."netrc_trusted" is casted to type "text" which is not the same as "json", the type of current target database column "public"."repos"."netrc_trusted".
2025-01-31T16:14:06.276036+01:00 LOG report summary reset
             table name     errors       rows      bytes      total time
-----------------------  ---------  ---------  ---------  --------------
                  fetch          0          0                     0.000s
        fetch meta data          0         19                     0.024s
      Drop Foreign Keys          0          0                     0.000s
-----------------------  ---------  ---------  ---------  --------------
              migration          0         25     0.7 kB          0.034s
                 agents          0         12     2.0 kB          0.032s
       pipeline_configs          0      22996   203.8 kB          0.128s
              pipelines          0       6928    10.8 MB          0.426s
            log_entries          0    1360606   240.7 MB          6.006s
                configs          0        472     1.2 MB          0.175s
                  perms          0         50     2.1 kB          0.190s
             registries          0         10     1.0 kB          0.229s
                  tasks          0          0                     0.268s
         server_configs          0          2     0.2 kB          0.292s
           redirections          0          1     0.0 kB          0.316s
                   orgs          0          3     0.1 kB          0.338s
                  repos          0         25    15.6 kB          0.419s
                  steps          0      29346     2.9 MB          0.205s
                secrets          0         43     4.1 kB          0.025s
                  users          0          2     3.8 kB          0.053s
                  crons          0          1     0.0 kB          0.056s
              workflows          0      10272   710.8 kB          0.173s
                 forges          0          1     0.1 kB          0.050s
-----------------------  ---------  ---------  ---------  --------------
COPY Threads Completion          0          4                     6.032s
        Reset Sequences          0         14                     0.033s
    Create Foreign Keys          0          0                     0.000s
       Install Comments          0          0                     0.000s
-----------------------  ---------  ---------  ---------  --------------
      Total import time          ✓    1430795   256.5 MB          6.065s
```
